integration/inspektor-gadget/trace_dns_test.go

```diff
@@ -80,9 +100,12 @@ func TestTraceDns(t *testing.T) {
 					Nameserver: dnsServer,
 					PktType:    "OUTGOING",
 					DNSName:    "fake.test.com.",
-					QType:      "A",
+					QType:      "AAAA",
```

Comment:
This is a mistake currently present in the tests. Thanks for catching it.

Reply:
+1